.agents/skills/working-with-devcontainers/SKILL.md

```
@@ -0,0 +1,256 @@
+**Version-stripped matching:** `installsAfter` values are always versionless per the spec (e.g. `ghcr.io/devcontainers/features/common-utils`). `Order` strips the version tag from registered feature IDs when resolving dependencies, so a feature registered as `…/common-utils:2` is correctly matched by an `installsAfter` entry of `…/common-utils`.
+
+```go
+ordered, err := features.Order(feats, metadata)
+// err is non-nil on cycle or missing metadata entry
+```
+
+## FeatureOptions.Merge
+
+Key normalisation: keys are uppercased and runs of non-alphanumeric characters replaced with `_`.
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)